docs/source/public/overview.rst

```diff
@@ -731,14 +733,30 @@ To upload the data, please follow these steps:
 
 1. Run the :ref:`post-simulation cleanup` script.
 2. Run the :ref:`data gathering script <gathering_of_results>`. The output of this step should be a compressed folder named ``results_data.zip``.
-3. Upload the compressed results folder to zenodo.
-
-  a) Go to `zenodo <https://zenodo.org/communities/openfreeenergy/records?q=&l=list&p=1&s=10&sort=newest>`_ to upload the data to the **Open Free Energy Zenodo Community**.
-  b) Click the `New upload` button on the top right.
-  c) Fill out all the relevant information. In the `Description` section, please report the OpenFE version that was used. You can get this information by running ``openfe --version`` in the commandline within the environment that you used to run the setup, simulations, and analysis of the data. In addition to that, please reference the `Industry Benchmark repo <https://github.com/OpenFreeEnergy/IndustryBenchmarks2024>`_ as
-     it contains all necessary information about the simulation details.
-  d) Add the compressed results folder (``results_data.zip``) to the zenodo upload.
-  e) Click the `Submit for review` button on the top right. We will then review the upload and publish it.
+3. Create a directory for each dataset, naming it after the dataset, e.g. ``jacs_bace``. Then, move the  ``results_data.zip`` file into the respective dataset folder. Please make sure to include both the set name (e.g. jacs or waterset) and the target name (e.g. bace) since some targets occur in multiple sets.
+
+.. code-block:: bash
+
+    mkdir jacs_bace
+    mv results_data.zip jacs_bace
+
```

Comment:
We should add something here about if its a private dataset make sure to copy in the experimental csv file into the folder.

Reply:
Added this!